noxfile.py

```diff
@@ -246,6 +251,7 @@ def emscripten(session: nox.Session, runner: str) -> None:
             )
 
         dist_dir = pyodide_artifacts_path
+    session.install("build")
     session.run("python", "-m", "build")
```

Comment:
Here too, `uv run -m build` can be used. But having to define `UV_PROJECT_ENVIRONMENT` makes it even less worth it than above.